docs/source/model_doc/bertforseqgeneration.rst

```diff
@@ -0,0 +1,44 @@
+BertForSeqGeneration
```

Comment:
dont see why `Seq` should be in the name. What other kind of generation might a confused person be thinking of?
dont feel strongly.

Reply:
changed it to `BertGeneration`